ld/scripttempl/elfarc.sc

```diff
@@ -264,6 +264,7 @@ cat <<EOF
   ${DATA_PLT-${BSS_PLT-${PLT}}}
   .jlitab       ${RELOCATING-0} :
   {
+    jlitab.o(.jlitab*)
```

Comment:
That got changed to jlitab*.o so it doesn't kill the linker if it doesn't exist. This file would be made by the user with jli_call_fixed and is in the default linker script to ensure fixed entries come before always entries in the case the two types are mixed.